ngraph/frontend/tensorflow/src/op/avg_pool.cpp

```diff
@@ -13,57 +13,58 @@ namespace frontend {
 namespace tf {
 namespace op {
 
-OutputVector TranslateAvgPoolOp(const NodeContext& node) {
-    Output<Node> ng_input = node.get_ng_input(0);
+OutputVector translate_avg_pool_op(const NodeContext& node) {
```

Comment:
`translate_avg_pool`. `op` is redundant

Reply:
op is ok, the same approach is used for all operations